templates/partials/footer.html

```diff
@@ -24,7 +24,7 @@
     <div class="row d-flex flex-column flex-lg-row justify-content-center">
         <div class="d-flex flex-column flex-lg-row funder-logo-container">
         <div class="w-25">
-            <a href="https://www.canada.ca/en/environment-climate-change.html"><img class="img-fluid" src="images/logos/funders/reverse/logo-eccc-en.png"></a>
+            <a href="https://www.canada.ca/en/environment-climate-change.html"><img class="first-funder-logo" src="images/logos/funders/reverse/logo-eccc-en.png"></a>
```

Comment:
Does this belong here? Or is this from another fix?

Reply:
This is from this morning.  It's a fix for the ECCC logo shrinking too small in mobile mode.